src/game/Tactical/Soldier_Profile.cc

```diff
@@ -608,6 +608,8 @@ SOLDIERTYPE* ChangeSoldierTeam(SOLDIERTYPE* const old_s, UINT8 const team)
 	new_s->bLastRenderVisibleValue = old_s->bLastRenderVisibleValue;
 	new_s->bVisible                = old_s->bVisible;
 	new_s->bCamo                   = old_s->bCamo;
+	new_s->bLevel                  = old_s->bLevel;
+	if(old_s->bLevel == 1) SetSoldierHeight(new_s, 58.0);
```

Comment:
There is no constant for this value by now. I took it from the Soldier_create.cc as it is used to put the right z-value. It is also used mulitple times in the whole code with a hardcoded 58.0 value.